ksqldb-functional-tests/src/test/resources/query-validation-tests/partition-by.json

```diff
@@ -279,11 +277,7 @@
       }
     },
     {
-      "comments": "Should be removed with rowpartition.rowoffset.enabled feature flag",
```

Comment:
I'm confused -- why is this comment being removed if the test is still here? Same question for below.

Reply:
If we remove the test(s), let's also make sure to remove any historic QTTs that may have been generated for them.